app/lib/server/methods/sendMessage.js

```diff
@@ -51,7 +51,6 @@ export function executeSendMessage(uid, message) {
 		fields: {
 			username: 1,
 			type: 1,
-			...!!settings.get('Message_SetNameToAliasEnabled') && { name: 1 },
```

Comment:
We should get the `name` field since the `sendMessage` uses it to set in `u`

Reply:
you're not wrong, BUT the name field stored on the message object is actually discarded and populated just before sending to clients via `normalizeMessagesForUser` function:

https://github.com/RocketChat/Rocket.Chat/blob/83240bc4f649c6364b77e448050a8c6c0a2f8c02/app/utils/server/lib/normalizeMessagesForUser.js#L14

that's why I decided to take it off.. maybe I should remove it from `sendMessage` as well, to prevent storing `name: null` on database and confusing everyone.